crates/llms/src/openai/mod.rs

```diff
@@ -87,6 +89,19 @@ impl Openai {
 
 #[async_trait]
 impl Chat for Openai {
+    fn as_sql(&self) -> Option<&dyn SqlGeneration> {
+        // Only use structured output schema for OpenAI, not openai compatible.
+        tracing::error!(
+            "self.client.config().api_base() ={}",
+            self.client.config().api_base()
+        );
```

Comment:
We shouldn't log debugging message as error output. Error outputs should be user-friendly

Reply:
Sorry, this was just me testing, never meant to be logged.